src/redux/initial-state/dev-lang.json

```diff
@@ -532,8 +532,8 @@
         "2012": "Component has failed!",
         "2500": "There was a problem while fetching project information",
         "2501": "Failed to change project information",
-        "2999": "clusterVisualization",
-        "3000": "Cluster visualization",
+        "2999": "Cluster Visualization",
+        "3000": "Cluster Visualization",
```

Comment:
Same text shouldn't repeat

Reply:
While I agree, I am too hesitant to edit it